RM_Vk/Helpers/VKNetworkService.swift

```diff
@@ -0,0 +1,102 @@
+// VKNetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// Перечисление методов для работы с сетью
+protocol VKNetworkServiceProvider {
+    func getFriends()
+    func getPhotos(for userID: Int)
+    func getGroups()
+    func getGroups(by searchQuery: String)
+}
+
+/// Класс для работы с сетью
+final class VKNetworkService: VKNetworkServiceProvider {
+    // MARK: - Private constants
+
+    private enum Constants {
+        static let baseURL = "https://api.vk.com/method/"
+        static let acessTokenParameter = "access_token"
+        static let versionParameter = "v"
+        static let versionValue = "5.131"
+        static let fieldsParameter = "fields"
+        static let friendFieldsValue = "nickname"
+        static let extendedParameter = "extended"
+        static let extendedValue = "1"
+        static let ownerIDParameter = "owner_id"
+        static let queryParameter = "q"
+
+        enum RequestMethod: CustomStringConvertible {
+            case getGroups
+            case getFriends
+            case searchGroups
+            case getPhotos
+
+            var description: String {
+                switch self {
+                case .getFriends:
+                    return "friends.get"
+                case .getGroups:
+                    return "groups.get"
+                case .searchGroups:
+                    return "groups.search"
+                case .getPhotos:
+                    return "photos.getAll"
+                }
+            }
+        }
+    }
+
+    // MARK: - Public methods
+
+    func getFriends() {
```

Comment:
```suggestion
    func fetchFriends() {
```
у нас может быть get, post запросы, лучше fetch